release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -1076,14 +1076,29 @@ In the following tables, features are marked with the following statuses:
 |Technology Preview
 
 |Hosted control planes for {product-title} on bare metal
-|Technology Preview
+|General Availability
 |General Availability
 |General Availability
 
 |Hosted control planes for {product-title} on {VirtProductName}
-|Technology Preview
 |General Availability
 |General Availability
+|General Availability
+
+|Hosted control planes for an ARM64 {product-title} cluster on AWS
+|Technology Preview
+|Technology Preview
+|Technology Preview
+
+|Hosted control planes for {product-title} on {ibm-power-title}
+|Technology Preview
+|Technology Preview
+|Technology Preview
+
+|Hosted control planes for {product-title} on {ibm-z-title}
+|Technology Preview
+|Technology Preview
+|Technology Preview
 
```

Comment:
@SNiemann15 Thanks so much for updating the hosted control planes table! We also need to add a row for non-bare metal agent machines:
```suggestion
|Hosted control planes for {product-title} using non-bare metal agent machines
|Not Available
|Technology Preview
|Technology Preview
```